lib/domUpdates.js

```diff
@@ -1,19 +1,37 @@
 const domUpdates = {
 
-  renderScoreboard(players) {
-    players.forEach((player) => {
+  renderScoreboard (players) {
+    players.forEach(player => {
       let ul = document.querySelector('.scoreboard');
-      let li = document.createElement('li');
-      ul.appendChild(li)
-      li.innerHTML = `Player: ${player.name}
-                      <br>
-                      Score: ${player.score}`;
+      let div = document.createElement('div');
+      ul.appendChild(div)
+      div.innerHTML = `<li>Player: ${player.name}</li>
+                      <li>Score: ${player.score}</li>`;
+    })
+  },
 
+  convertCategoryNames(camelCaseNames) {
+    return camelCaseNames.map(name => {
+      let result = name.replace( /([A-Z])/g, " $1" );
+      let finalResult = result.charAt(0).toUpperCase() + result.slice(1);
+      return finalResult;
     })
-  }
+  },
+
+  renderGameboard (categories) {
+    let convertedNames = this.convertCategoryNames(categories);
+    let gameBoard = document.querySelector('.game-board');
+    convertedNames.forEach(currentName => {
+
+      let ul = document.createElement('ul');
+      ul.innerHTML = `<li class="category">${currentName}</li>`
+      gameBoard.appendChild(ul);  
```

Comment:
Are you intending to append a `ul` item for each player name? With a single `li` item in each unordered list? Normally you'd create a single `ul` element that contained an `li` for each name. (You won't usually see unordered lists that only have a single `li` element in them, which is what it looks like this block of code is generating at the moment.)